lib/versioning/VersionID.js

```diff
@@ -40,16 +40,21 @@ function padRight(value, template) {
     return `${value}${template}`.slice(0, template.length);
 }
 
-// site identifier, like PARIS, TOKYO; will be trimmed if exceeding max length
-const SITE_ID = padRight(process.env.SITE_ID, TEMPLATE_SITE);
-
 // constants for max epoch and max sequential number in the same epoch
 const MAX_TS = Math.pow(10, LENGTH_TS) - 1; // good until 16 Nov 5138
 const MAX_SEQ = Math.pow(10, LENGTH_SEQ) - 1; // good for 1 billion ops
 
-// the earliest versionId, used for versions before versioning
-const VID_INF = (padLeft(MAX_TS, TEMPLATE_TS) +
-                 padLeft(MAX_SEQ, TEMPLATE_SEQ) + SITE_ID);
+/**
+ * Generates the earliest versionId, used for versions before versioning
+ *
+ * @param {string} replicationGroupToken - replication unit identifier
+ * @return {string} version ID for versions before versionig
+ */
+function generateDummyVersionId(replicationGroupToken) {
```

Comment:
This function should rather be called `generateEarliestVersionId`, dummy sounds to me like it's only for testing (but I'm not a native English speaker!)